pkgs/top-level/all-packages.nix

```diff
@@ -12375,6 +12375,8 @@ with pkgs;
 
   re-isearch = callPackage ../applications/search/re-isearch { };
 
+  reason-shell = callPackage ../tools/misc/reason-shell { };
```

Comment:
https://github.com/NixOS/rfcs/pull/140 was merged already. Nonetheless, and in light of https://github.com/NixOS/rfcs/pull/146, I believe this pertains to `applications/science/misc`.